app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -648,18 +648,20 @@ export default Component.extend({
     }
   },
 
-  _selectText(from, length) {
+  _selectText(from, length, opts = { scroll: true }) {
     next(() => {
```

Comment:
while we are at it, I'm unsure why we have a next and not a schedule("afterRender") here